### ISSUE TITLE
The crawling process fails when the page title contains the `|` character running PW AI Bot on Windows

### ISSUE THREAD

**Issue by marcel-veselka** [Wopee-io/playwright-ai-bot-demo#2]
Running playwright-ai-bot on Windows on version 1.4.1

Getting the following error:

```shell
✘ Error processing https://dronjo.wopee.io/buy.html: Error: ENOENT: no such file or directory, open 'C:\Users\lenca\OneDrive\Plocha\Marcel\playwright-ai-bot-demo\playbot-data\screens\7-Checkout-DJI-4-Pro-—-dronjo-|-your-one-stop-shop-for-the-best-drones.png'
at async open (node:internal/fs/promises:639:25)
at async Object.writeFile (node:internal/fs/promises:1212:14)
at async Page.screenshot (C:\Users\lenca\OneDrive\Plocha\Marcel\playwright-ai-bot-demo\node_modules\playwright-core\lib\client\page.js:526:7)
at async saveScreenshot (file:///C:/Users/lenca/OneDrive/Plocha/Marcel/playwright-ai-bot-demo/node_modules/playwright-ai-bot/dist/utils/crawl.js:60:5)
at async processLinks (file:///C:/Users/lenca/OneDrive/Plocha/Marcel/playwright-ai-bot-demo/node_modules/playwright-ai-bot/dist/utils/crawl.js:41:13)
at async crawl (file:///C:/Users/lenca/OneDrive/Plocha/Marcel/playwright-ai-bot-demo/node_modules/playwright-ai-bot/dist/utils/crawl.js:12:22)
at async Command. (file:///C:/Users/lenca/OneDrive/Plocha/Marcel/playwright-ai-bot-demo/node_modules/playwright-ai-bot/dist/commands/crawl.js:54:29) {
errno: -4058,
code: 'ENOENT',
syscall: 'open',
path: 'C:\Users\lenca\OneDrive\Plocha\Marcel\playwright-ai-bot-demo\playbot-data\screens\7-Checkout-DJI-4-Pro-—-dronjo-|-your-one-stop-shop-for-the-best-drones.png'
}
```

**Comment by marcel-veselka** [comment]
🎉 **Fixed!** 🎉

The issue should now be all fixed in version `1.4.2`! 🚀

**Comment by marcel-veselka** [comment]
🎉 Retest successful! 
Closing this one out. ✅
